trifid/templates/deployment.yaml

```diff
@@ -60,10 +60,20 @@ spec:
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-    {{- with .Values.affinity }}
       affinity:
-        {{- toYaml . | nindent 8 }}
-    {{- end }}
+        podAntiAffinity:
```

Comment:
Why is this hardcoded in the template now? Why not make it default in the values.yaml? This way it's not customizable at all.